### PR TITLE
feat(runner): support hooks with multiple commands

### PR DESCRIPTION
**Pull request by eranshmil**
Added support to run hooks with multiple commands, except for old commands.

Related issues: #330 #319

### CODE REVIEW

**Comment by nunofgs** [comment]
Need this! When is it going to be merged? 🙃

**Comment by joseSantacruz** [comment]
+1

File: src/runner/index.ts [hunk 1]
```diff
@@ -51,9 +51,18 @@ export default async function run(
...
+    if (commands) {
+      if (!Array.isArray(commands)) {
+        commands = [commands]
+      }
+
+      commands.forEach((command: string) => {
+        console.log(
+          `husky > ${hookName} > ${command} (node ${process.version})`
+        )
+        execa.shellSync(command, { cwd, env, stdio: 'inherit' })
```
Comment (datashaman) [inline src/runner/index.ts]: What if a command fails?

Comment (eranshmil) [reply src/runner/index.ts]: Yeah, you're right.
I'll change that if the PR will be relevant.

**Comment by G-Rath** [comment]
Love this! Anything I can do to help get it across the line? (I was all set to fork to implement this before I saw this PR 😂)

Also, I'm not sure who's managing it, but the [`huskyrc` schema](http://json.schemastore.org/huskyrc) should be updated before this is released :D

**Comment by pke** [comment]
Is the missing schema update blocking this from being merged?

**Pull request closed without merging**